bin/audit

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+## Runs audit on all lerna packages
+## By default, it just runs the audit. To fix issues, pass 'audit' argument
+## i.e.
+##    ./bin/audit       # Runs the audit
+##    ./bin/audit fix   # Attempts to auto-fix issues
+
+homeDir="$(pwd)"
+arg=$1
+
+# Check requires packages
+if ! command -v /usr/local/bin/ncu &> /dev/null ; then
```

Comment:
Should we look for command available on path vs looking under `/usr/local/bin` ?
```suggestion
if ! command -v ncu &> /dev/null ; then
```

Reply:
`ncu` can also be `NCU Real Time Weather`. This is intended to run only on the developer computer, so we can assume all binaries would be under `/usr/local/bin`

Reply:
Offline chat: intended to run on local machine and packages installed using `-g` which puts stuff in /usr/local.